addons/hr_holidays/static/src/components/partner_im_status_icon/partner_im_status_icon.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
-    <t t-inherit="mail.messaging.component.PartnerImStatusIcon" t-inherit-mode="extension">
+    <t t-inherit="mail.mail/static/src/components/partner_im_status_icon/partner_im_status_icon.xml" t-inherit-mode="extension">
```

Comment:
This is the only thing so far that I don't like. Would be best if server handled the first `/` as a separator same as a `.`.

Reply:
Suggestion to allow multiple templates per file: 

> mail/static/src/components/partner_im_status_icon/partner_im_status_icon.xml:**ClassName**

Even more verbose though, might be optional if there is only one template in the file?